Source/websocket/JSONRPCLink.h

```diff
@@ -460,14 +460,67 @@ namespace Thunder {
             using PendingMap = std::unordered_map<uint32_t, Entry>;
             using InvokeFunction = Core::JSONRPC::InvokeFunction;
 
+            class Handler {
+                public:
+                using HandlerMap = std::unordered_map<string, InvokeFunction>;
+
+                public:
+                    Handler():_adminLock(),_invokeMap(){}
+                    Handler(const Handler&) = delete;
+                    Handler& operator=(const Handler&) = delete;
```

Comment:
Also delete the move  constructor and the move assignment operator (latest coding standards)